src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,55 @@
+.stopwatch {
+  @include circle($size);
+
+  box-sizing: border-box;
+  position: relative;
+  background-color: $color-grey;
+  border: $borderWidth solid $color-dark;
+
+  &__second {
+    width: $secondWidth;
+    height: $secondHight;
+    background-color: $color-red;
+    border-radius: 3px;
+
+    position: absolute;
+    left: getShift($size, $borderWidth, $secondWidth);
+    top: getShiftTop($size, $borderWidth, $secondHight);
+
+    transform-origin: bottom;
+    animation: move 60s steps(60, end) infinite;
+  }
+
+  &__minute {
+    width: $minuteWidth;
+    height: $minuteHight;
+    background-color: $color-black;
+    border-radius: 6px;
+
+    position: absolute;
+    top: getShiftTop($size, $borderWidth, $minuteHight);
+    left: getShift($size, $borderWidth, $minuteWidth);
```

Comment:
Pay attention to CSS properties order
https://9elements.com/css-rule-order/
![image](https://user-images.githubusercontent.com/96409563/226163873-8d4fd67a-5b55-4a4c-8fa6-bdba2a6089ab.png)